jax/experimental/key_reuse/_core.py

```diff
@@ -114,6 +116,17 @@ def __repr__(self):
     return f"Forward({self.in_idx}, {self.out_idx})"
 
 
+def _format_frame(frame: inspect.FrameInfo):
```

Comment:
Use exception chaining instead of formatting frames and appending to messages? Following what we do elsewhere.

Reply:
I'm not sure we can do that, because the original stack trace is not associated with any exception.

Reply:
OK, done